### DISCUSSION TITLE
Add "Fast Forward" option (similar to DosBox-X turbo mode)

### DISCUSSION THREAD

**Discussion by viti95** [86Box/86Box#3235] [Ideas]
**Is your feature request related to a problem? Please describe.**
Some software spents lot's of time installing on certain systems. Having a fast forward button makes the emulation run as fast as possible (basically unthrottled), lowering the installation times. This is also beneficial for long run benchmarks.

**Describe the solution you'd like**
The idea is to copy DosBox-X Turbo mode, which makes the emulation run as fast as possible. 

**Describe alternatives you've considered**
Other option would be to be able to change the CPU frequency in real-time, but this is not ideal when trying to execute batched benchmarks. I've made a script that evaluates FastDoom performance with 210 automated tests. This requires needs long time to execute on slower systems (for example a 386SX can spend more than 5 hours to execute everything).

**Additional context**
![imagen](https://user-images.githubusercontent.com/8323882/227191523-193489d2-c6e2-45b8-ace8-c7f37c439473.png)

**Comment by dhrdlicka** [comment]
I don't think this is possible without a substantial rewrite of the emulator core.
